Assets/AppCenter/Editor/BaseDependency.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using UnityEditor;
+using UnityEngine;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+public abstract class BaseDependency
+{
+    protected Type versionHandler;
+    protected Type playServicesSupport;
```

Comment:
```suggestion
    protected Type PlayServicesSupport { get; set; }
```
(Same comment as above)